samples/config.default.js

```diff
@@ -1,36 +1,57 @@
-// User-specific configuration
-exports.voice = 'en-US_MichaelVoice';
-exports.conversationWorkspaceId = ''; // replace with the workspace identifier of your conversation
+// User-specific CONFIGURATION
+exports.config = {}
+exports.config.voice = {"gender": "female", "language": "en-US"};
+exports.config.conversationWorkspaceId = ''; // replace with the workspace identifier of your conversation
 
-// Create the credentials object for export
-exports.credentials = {};
+// Card and Device found in command arecord -l of your hardware. In general 0,0 or 1,0 or 0,1
+// If bot is closing without wait for listening you maybe have to change that parameter
+exports.config.microphone = "plughw:0,0";
+// Attention word that bot are waiting for
+exports.config.attentionWord = 'TJ';
+// To see log messages
+exports.config.verboseLogging = false;
 
-// Watson Conversation
-// https://www.ibm.com/watson/developercloud/conversation.html
-exports.credentials.conversation = {
-    password: '',
-    username: ''
+exports.config.ledPin = 8;
+exports.config.servoPin = 7;
```

Comment:
It was moved from tjbot.js with these exactly values:
https://github.com/ibmtjbot/tjbotlib/blob/master/lib/tjbot.js